### DISCUSSION TITLE
Q.A.

### DISCUSSION THREAD

**Discussion by aiwei169** [pyswmm/pyswmm#345] [Q&A]

**Comment by aiwei169** [comment]
Hi guys, I think you were doing a very great job in this project. It is very meaningful to the water Analytics career. PySWMM has helped me a lot.
I got some problems that, do you guys have some real-data to prove or test the result from this python package?

**Comment by bemcdonnell** [comment]
@aiwei169 can you clarify what you are asking for? Do you want to see regression testing information to prove that our results are the same as USEPA SWMM?

**Comment by dickinsonre** [comment]
Maybe he means calibration data?  Also an issue with EPA SWMM?
